src/deployment/cd.md

```diff
@@ -193,6 +193,151 @@ secrets in pull requests that you accept and merge.
          * `cd android` or `cd ios`
          * `bundle exec fastlane [name of the lane]`
 
+## Xcode Cloud
+
+[Xcode Cloud][] is a continuous integration and delivery service for building,
+testing and distributing apps and frameworks for Apple platforms.
+
+### Requirements
+
+* Xcode Version 13.4.1 or higher.
+* Be enrolled in the [Apple Developer Program][].
+
+### Custom Build Script
+
+Xcode Cloud recognizes three different [custom build scripts][] that can be
+used to perform additional tasks at a designated time.
+
+{{site.alert.note}}
+The temporary build environment that Xcode Cloud uses includes tools that are
+part of macOS and Xcode — for example, Python — and additionally Homebrew to
+support installing third-party dependencies and tools.
```

Comment:
```suggestion
  The temporary build environment that Xcode Cloud uses includes tools that are
  part of macOS and Xcode&mdash;for example, Python&mdash;and additionally Homebrew to
  support installing third-party dependencies and tools.
```
Flutter doesn't use Homebrew. Is this an issue?

Reply:
> Flutter doesn't use Homebrew

Indeed, though CocoaPods is required when using flutter plugins for iOS or macOS